devtools/conda-envs/openmm.yaml

```diff
@@ -1,13 +1,12 @@
 name: test
 channels:
   - conda-forge
-  - omnia
 dependencies:
-  - conda-forge::rdkit
-  - omnia::openmm
-  - omnia::openforcefield
-  - omnia::openforcefields
-  - omnia::openmmforcefields
+  - rdkit
+  - openmm
+  - openff-toolkit
+  - openff-forcefields
+  - openmmforcefields
```

Comment:
The openff PRs are merged, so I retriggered CI. It's balking on `openmmforcefields` now. Does it need a channel change?

Reply:
Thanks @loriab - we're just waiting on https://github.com/conda-forge/staged-recipes/pull/13448 now and once a C-F release of `openmmforcefields` is done the checks should all come back green!

Reply:
@loriab the package should now be on C-F - can you please re-trigger the CI?

Reply:
it works! congrats on getting everything over to c-f.

I'm working on converting the travis tests to GHA, so I may possibly be pinging you again. And I notice the openmm+geometric tests aren't getting run, so some testing lanes may need adjusting.